CHANGELOG.md

```diff
@@ -5,6 +5,35 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Configuration options required for establishing a connection with the central
+  management server must be specified directly via command-line interface.
+  The following options and arguments are required:
+  - `--cert`: Specifies the certificate file for the current module.
+  - `--key`: Specifies the private key file for the certificate.
+  - `--ca-certs`: Specifies the CA certificate files. Multiple CA certificates
+    can be provided by using this option multiple times.
+  - The Manager server's name and address must be provided as a command-line
+    argument in the format `<server_name>@<server_ip>:<server_port>`.
+- The configuration file, previously provided as a positional command-line
+  argument, must now be specified using the `-c` option. This change allows for
+  additional command-line arguments to be used alongside the configuration file.
+  However, providing a local configuration file is optional.
+- If `log_dir` is not specified in the configuration file or the log file cannot
+  be created, logs will be printed to stdout.
+- Default last_timestamp_data file: /usr/local/aice/conf/time_data.json
```

Comment:
will remove the above line according to the [comment](https://github.com/aicers/crusher/pull/114#discussion_r1825610290).